### PR TITLE
Add link to service unavailable sass file.

### PR DESCRIPTION
**Pull request by MancunianSam**
We're now using the same custom css in the tdr-service-unavailable repo.
I've added a link here so we replicate any changes in that repo.

### CODE REVIEW

File: npm/css-src/sass/_header.scss [hunk 1]
```diff
@@ -1,4 +1,5 @@
 // Stylesheet also used in tdr-auth-server repo: https://github.com/nationalarchives/tdr-auth-server
+// Stylesheet also used in tdr-service-unavailable repo: https://github.com/nationalarchives/tdr-service-unavailable/blob/master/css-src/sass/_header.scss
 // Any changes should also be added to tdr-auth-server repo version
```
Comment (TomJKing) [inline npm/css-src/sass/_header.scss:3]: Reword the comment to: "Any changes should also be added to the tdr-auth-server and tdr-service-unavailable repo versions."

**Review by TomJKing — COMMENTED** [review]
Looks good. Just one minor suggestion.

File: npm/css-src/sass/_header.scss (hunk 1 above)
Comment (MancunianSam) [reply npm/css-src/sass/_header.scss]: Good point. I've changed it.

**Review by TomJKing — APPROVED** [review]
Looks good.

**Pull request merged**